package.json

```diff
@@ -30,17 +30,17 @@
   ],
   "scripts": {
     "postinstall": "yarn compile && yarn run readme",
-    "test": "yarn workspaces run test",
+    "test": "yarn workspaces foreach -v run test",
```

Comment:
why verbose?

Reply:
Well it looks better in the CLI, no other reason than that

Reply:
As per documentation it `Prefixes each output line with the name of the originating workspace` but it also adds colors to differentiate the workspaces 🌈